src/helpers/figures/charts/chart_ui_common.ts

```diff
@@ -151,6 +151,7 @@ export function getDefaultChartJsRuntime(
       labels: labels.map(truncateLabel),
       datasets: [],
     },
+    platform: DomPlatform,
```

Comment:
You can simply remove `Require` line 104, and `platform` will be correctly added by Chart.JS

Reply:
### 
```suggestion
    platform: undefined as unknown as typeof BasePlatform, // This key is optional and will be set by chart.js
```